test/cclq-cram/merge-query.t

```diff
@@ -49,11 +57,11 @@ Merge and queries
   1 =
   12341234 =
   
-  someval =
+  someval
   
-  baz =
+  baz
   
 Pipe queries
   $ cat sample2.ccl | cclq this | cclq bar
-  baz =
+  baz
```

Comment:
it makes the queries a bit better because you get the actual value of the config, not a ccl document that encodes the actual value via a singleton key-nullvalue